client/lib/api/content/content_store.dart

```diff
@@ -134,5 +134,10 @@ abstract class _ContentStore with Store implements Updateable {
         symptomPoster = v;
       })
     ]);
+
+    unawaited(
+      UserPreferences()
```

Comment:
Does this wait on the CaseStats being updated as well? At the moment, it looks as though the yaml could updated... at which point the client doesn't want about old data, even though the CaseStats could be stale? It needs to track the oldest data from either system.

Reply:
This currently looks at only the overall last time the stats were updated.  In the next round I will extend it to each individual Store.

Reply:
Its also worth noting that when the stats store and content store update, they update by calling the overall update function.

[Stats Store](https://github.com/WorldHealthOrganization/app/blob/f3080c4c7ccf5cbaf32a12a26d76a672c7848e57/client/lib/api/stats_store.g.dart#L74)

[Content Store](https://github.com/WorldHealthOrganization/app/blob/29ec7ceba52e7f90ed9243071fbda987a892b204/client/lib/api/content/content_store.g.dart#L160)